scripts/crontab/crontab.tpl

```diff
@@ -15,6 +15,7 @@ HOME=/tmp
 10 * * * * %(z_cron)s update_blog_posts
 15 * * * * %(django)s send_pending_rejection_last_warning_notifications
 20 * * * * %(z_cron)s addon_last_updated
+30 * * * * %(z_cron)s process_blocklistsubmissions
```

Comment:
Could be more frequent but I don't know how often they're be created and how timely the block will need to be 🤷

Reply:
Once per hour is probably fine IMHO.